docs/Operation-Architecture/Lifecycle_Management/lifecycle_expiration_policies.rst

```diff
@@ -1,20 +1,20 @@
-=============================
-Lifecycle Expiration Policies
-=============================
-Cloud users can apply lifecycle expiration rules (specified in Amazon’s
-`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
-to buckets managed through Zenko. These rules are triggered after a defined
-time has passed since the object’s creation.
-
-Zenko supports expiration of versioned or non-versioned objects, when a
-defined number of days has passed since those objects’ creation. This enables
-automatic deletion of older versions of versioned objects to reclaim storage
-space. Zenko also supports triggering expiration of current versions on a date
-from which a rule applies. This is currently feasible using the S3 API, but not
-with Orbit. Using Zenko from the command line or from Orbit, you can expire the
-current version of an object with a separate rule. For versioned buckets, add
-a delete marker on an event, such as when a user deletes an object without first
-specifying a version ID.
+Lifecycle Expiration Policies
+=============================
+
+Cloud users can apply lifecycle expiration rules (specified in Amazon’s
+`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
+to buckets managed through Zenko. These rules are triggered after a defined
+time has passed since the object’s creation.
+
+Zenko supports expiration of versioned or non-versioned objects, when a
+defined number of days has passed since those objects’ creation. This enables
+automatic deletion of older versions of versioned objects to reclaim storage
+space. Zenko also supports triggering expiration of current versions on a date
+from which a rule applies. This is currently feasible using the S3 API, but not
+with Orbit. Using Zenko from the command line or from Orbit, you can expire the
+current version of an object with a separate rule. For versioned buckets, add
```

Comment:
I'm not clear on the meaning of this sentence "For versioned buckets, add a delete marker on an event...", I think it means "For versioned buckets, *lifecycle* adds a delete marker automatically when a rule triggers, such as when a user ..."

Reply:
made change.